compiler/tests/org/jetbrains/kotlin/cli/DeterministicOutputTest.kt

```diff
@@ -63,6 +63,22 @@ class DeterministicOutputTest : TestCaseWithTmpdir() {
         assertNoTimestampsAreReset(jar)
     }
 
+    /**
```

Comment:
Let's rename the test class from `DeterministicOutputTest` to something like `JarOutputTest`, now that it tests not only the timestamps in the jar file, but also the absence of module-info

Reply:
I was thinking the name `DeterministicOutputTest` is suitable for it's new mission, it seems I was wrong about this.
I will rename it very soon.